rekcurd_dashboard/apis/api_evaluation.py

```diff
@@ -31,26 +32,56 @@
     'status': fields.Boolean(required=True),
     'evaluation_id': fields.Integer(required=True, description='ID of uploaded data')
 })
+evaluation_params = evaluation_api_namespace.model('Evaluation', {
+    'evaluation_id': fields.Integer(
+        readOnly=True,
+        description='Evaluation ID.'
+    ),
+    'checksum': fields.String(
+        readOnly=True,
+        description='Checksum for file content'
+    ),
+    'application_id': fields.String(
+        readOnly=True,
+        description='Application ID.'
+    ),
+    'description': fields.String(
+        readOnly=True,
+        description='Description of evaluation data'
+    ),
+    'data_path': fields.String(
+        readOnly=True,
+        description='Evaluation file path'
+    ),
+    'register_date': DatetimeToTimestamp(
+        readOnly=True,
+        description='Register date'
+    )
+})
 
 
 @evaluation_api_namespace.route('/projects/<int:project_id>/applications/<application_id>/evaluations')
 class ApiEvaluations(Resource):
     upload_parser = reqparse.RequestParser()
-    upload_parser.add_argument('file', location='files', type=FileStorage, required=True)
+    upload_parser.add_argument('filepath', location='files', type=FileStorage, required=True)
+    upload_parser.add_argument('description', location='form', type=str, required=True)
 
     @evaluation_api_namespace.expect(upload_parser)
     @evaluation_api_namespace.marshal_with(eval_data_upload)
     def post(self, project_id: int, application_id: str):
         """update data to be evaluated"""
         args = self.upload_parser.parse_args()
-        file = args['file']
+        file = args['filepath']
+        description = args['description']
         checksum = HashUtil.checksum(file)
 
         evaluation_model = db.session.query(EvaluationModel).filter(
             EvaluationModel.application_id == application_id,
             EvaluationModel.checksum == checksum).one_or_none()
         if evaluation_model is not None:
-            return {"status": True, "evaluation_id": evaluation_model.evaluation_id}
+            raise RekcurdDashboardException(
+                'the file already exists. ID: {}, Description: {}'.format(
+                    evaluation_model.evaluation_id, evaluation_model.description))
```

Comment:
I changed return 400 instead of 200 in this case 
because it is weird to display as Success but  nothing is added to the list page when uploading same content file.

Reply:
I just noticed that in this way, there is no way to know evaluation_id of existing file contents as JSON response.
So I will change back to return 200 and add message to JSON response

Reply:
I did it: https://github.com/rekcurd/dashboard/pull/103/commits/a631ecada3ca7181405bd2e7fcbb46df97c420c2